src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -503,13 +506,18 @@
 
 (rf/reg-event-fx :wallet/end-transaction-flow
  (fn [{:keys [db]}]
-   (let [address (get-in db [:wallet :current-viewing-account-address])]
-     {:fx [[:dispatch [:wallet/navigate-to-account-within-stack address]]
+   (let [address        (get-in db [:wallet :current-viewing-account-address])
+         from-settings? (get-in db [:wallet :ui :send :from-settings?])]
+     {:fx [(when from-settings?
+             [:dispatch [:pop-to-root :shell-stack]])
+           (if from-settings?
+             [:dispatch-later [{:ms 600 :dispatch [:wallet/navigate-to-account address]}]]
```

Comment:
Any reason why `600ms` was chosen as the wait time?

Reply:
@BalogunofAfrica we want to delay `navigate-to-account` until `pop-to-root` completes. I tried using less time but it did not work.

Reply:
I ask because we have a case like this in this PR also:

https://github.com/status-im/status-mobile/pull/20657#discussion_r1666951709

And `1ms` seemed to work, so it gets confusing determining the arbitrary time to wait before dispatching

Reply:
@BalogunofAfrica that's something different which you mentioned (it's just delaying navigation for re-frame to update db). Here we need pop-to-root navigation to execute first (a screen navigation takes at least 300ms), then other background tasks happening may cause further delay.

Reply:
You are right that they are different context. My concern is just having an arbitarary number for waiting. Also thinking if this would work the same for this context no matter where it is used. Maybe we can have a constant for these delays, not really sure of the best way to handle them.

Reply:
I don't think this `dispatch-later` is a clean approach :thinking: 

We should create a mechanism to navigate a path of routes in a general way so that we don't need to manually measure the ms. Also, I'm not sure if this time (`600`) is constant across different devices or platforms :thinking: 

@OmarBasem  Do you think this issue is a correct place to explore a more general solution?

@J-Son89 I think you've done something similar to navigate, wdyt?

Reply:
@ulisesmac we checked this with Pedro in the last stand up and he said we can reset the shell to root when starting send flow from settings (demo: https://github.com/status-im/status-mobile/pull/20676#discussion_r1671977017). Now am using a timeout of 400ms before starting the send flow as poping to root is a navigation that takes time to execute

Reply:
Great!, thanks @OmarBasem